docs/user/alert-definitions.rst

```diff
@@ -192,7 +192,8 @@ Example JSON HipChat configuration:
    [
       "send_hipchat()",
       "send_hipchat(room='#incidents', color='red')",
-      "send_hipchat(room='#incidents', token='your-token')"
+      "send_hipchat(room='#incidents', token='your-token')",
+      "send_hipchat(room='#incidents', token='your-token', notify='True')"
```

Comment:
Without quotes it throws: 
zmon_worker_monitor/zmon_worker/tasks/notacelery_task.py", line 946, in safe_eval 
NameError: name 'True' is not defined
 
Quotes make it work.